business/istio_config.go

```diff
@@ -844,10 +844,9 @@ func checkType(types []string, name string) bool {
 	return false
 }
 
-func ParseIstioConfigCriteria(namespace, objects, labelSelector, workloadSelector string) IstioConfigCriteria {
+func ParseIstioConfigCriteria(namespace, objects, labelSelector, workloadSelector string, allNamespaces bool) IstioConfigCriteria {
```

Comment:
not blocking: I suppose this could have just driven off of `namespace == ""` and avoided having both a `namespace` and `allNamespaces` parameter.

Reply:
There is a case where there is no namespace but no "all namespaces flag" it should return an empty list.
This flag is to cover that case.